modules/git/repo_attribute.go

```diff
@@ -126,7 +121,7 @@ type CheckAttributeReader struct {
 	cancel      context.CancelFunc
 }
 
-// Init initializes the cmd
+// InitWithConfigSync initializes the cmd
```

Comment:
```suggestion
// Init initializes the CheckAttributeReader
```

Reply:
Oops .... GoLand is not smart enough ...........